crates/stateless/Cargo.toml

```diff
@@ -0,0 +1,42 @@
+[package]
+name = "reth-stateless"
+version.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+license.workspace = true
+homepage.workspace = true
+repository.workspace = true
+exclude.workspace = true
+
+[lints]
+workspace = true
+
+[dependencies]
+# alloy
+alloy-primitives.workspace = true
+alloy-rlp.workspace = true
+alloy-trie.workspace = true
+alloy-rpc-types-debug.workspace = true
+alloy-consensus.workspace = true
+alloy-eips = { workspace = true }
+
+# reth
+reth-ethereum-consensus.workspace = true
+reth-primitives.workspace = true
```

Comment:
I'd like to avoid this because this crate is being phased out: #13873 

use can use `reth-primitives-traits` and if required the eth primitives for the eth types

Reply:
sounds good -- will replace reth-primitives with reth-primitives-traits